apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-container.tsx

```diff
@@ -0,0 +1,55 @@
+import { AsyncRenderer, Splash } from '@vegaprotocol/ui-toolkit';
+import { gql, useQuery } from '@apollo/client';
+import type { DealTicketQuery } from '@vegaprotocol/deal-ticket';
+import { DealTicketManager } from '@vegaprotocol/deal-ticket';
+import { t } from '@vegaprotocol/react-helpers';
+import { DealTicketSteps } from './deal-ticket-steps';
+
+const DEAL_TICKET_QUERY = gql`
+  query DealTicketQuery($marketId: ID!) {
+    market(id: $marketId) {
+      id
+      decimalPlaces
+      state
+      tradingMode
+      tradableInstrument {
+        instrument {
+          product {
+            ... on Future {
+              quoteName
+            }
+          }
+        }
+      }
+      depth {
+        lastTrade {
+          price
+        }
+      }
+    }
+  }
+`;
+
+interface DealTicketContainerProps {
+  marketId: string;
+}
+
+export const DealTicketContainer = ({ marketId }: DealTicketContainerProps) => {
+  const { data, loading, error } = useQuery(DEAL_TICKET_QUERY, {
+    variables: { marketId },
+  });
+
+  return (
+    <AsyncRenderer<DealTicketQuery> data={data} loading={loading} error={error}>
+      {data && data.market ? (
+        <DealTicketManager market={data.market}>
```

Comment:
suggestion (non-blocking) This is almost identical to deal-ticket-container in the lib/deal-ticket. Perhaps we add a render prop so we can provide different form components to be rendered but allow the 'container' to make the fetch request and the 'manager' to handle order state..

Something like:
```
// simple-trading-app
<DealTicketContainer
  render={({ market, submit }) => <DealTicketSteps market={market} submit={submit} />}
/>
// trading
<DealTicketContainer
  render={({ market, submit }) => <DealTicket market={market} submit={submit} />}
/>

```